### ISSUE TITLE
bad links to resources in full html

### ISSUE THREAD

**Issue by litlfred** [dhis2/DeansDashboardDocs#5]
In the chunked html, resources are linked like:
    http://dhis2.github.io/DeansDashboardDocs/en/html/resources/images/import_export/display_import_summary.png

but for the full html, they are linked like:
   http://dhis2.github.io/DeansDashboardDocs/en/resources/images/implementation_guide/dhis_platform.png

So we are missing an "html"

**Comment by jason-p-pickering** [comment]
Not sure  this is a "real" issue. We have done it this way I think so that we do not need both sets of images, at least that is the idea. Meaning, on the final deployment, the chunked HTML and non-chunked HTML reference the same set of images, rather than two sets of the same images.

**Comment by litlfred** [comment]
I completely agree about using the same resource.  It’s just (for whatever reason) the link is wrong in the full html.  This could be b/c I moved the target location of the full .html
Cheers,
-carl
